NanoProd/crab/Run3_2022_postEE/DY.yaml

```diff
@@ -5,4 +5,21 @@ config:
     storeFailed: True
 
 DYJetsToLL_M-50: /DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EEMiniAODv3-forPOG_124X_mcRun3_2022_realistic_postEE_v1-v3/MINIAODSIM
+DYTo2L_MLL-4to50_Poisson60KeepRAW: /DYToLL_M-4To50_TuneCP5_13p6TeV-pythia8/Run3Summer22EEMiniAODv3-Poisson60KeepRAW_124X_mcRun3_2022_realistic_postEE_v1-v2/MINIAODSIM
```

Comment:
for all 2022 MC configs drop `KeepRAW` in the name - it doesn't bring any useful physics information. E.g. `DYTo2L_MLL-4to50_Poisson60KeepRAW` -> `DYTo2L_MLL-4to50_Poisson60` etc.